### PR TITLE
Order external urls

### PR DESCRIPTION
**Pull request by marusak**
Fixes abrt/faf#598

### CODE REVIEW

File: src/webfaf/templates/_helpers.html [hunk 1]
```diff
@@ -371,11 +371,14 @@
...
-    </a>
-  </li>
+{% for report_url in urls %}
+    {% for url_obj in report_url %}
+      <li><a
+        href="{{ url_obj.url }}">{{ url_obj.url }}
+        </a>
+      </li>
+    {% endfor %}
+    <li class="empty-line"></li>
```
Comment (mkutlak) [inline src/webfaf/templates/_helpers.html]: Maybe do something like this:
\<li class="empty-line"\> \<hr /\>  \</li\>

li.empty-line hr {margin: 0.5em 0;} 
li.empty-line:last-child { display: none; }

Comment (marusak) [reply src/webfaf/templates/_helpers.html]: hmm, this adds line. It stretches across the whole available space. I don't this it looks good. But I let it for others to decide. Ideas anyone?
after

Comment (mkutlak) [reply src/webfaf/templates/_helpers.html]: Yeah, it looks weird when the URLs are short. This is how it looks like with normal URLs. fafpr

**Comment by marusak** [comment]
Updated. The line is now 50% of the width. And I also added tooltips. 
report
problem_full

**Comment by mkutlak** [comment]
The tooltip could say something like this:
"The URLs are grouped by reports. Newest URL is always at the top."
Nothing else comes to my mind.

Otherwise it looks good and can be merged.

**Comment by marusak** [comment]
Updated.

**Comment by mkutlak** [comment]
LGTM. Thank you!

**Pull request merged**